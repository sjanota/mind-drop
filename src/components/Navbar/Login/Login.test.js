import React from 'react';
import renderer from 'react-test-renderer';
import Login from "./Login";
import {Auth0Context} from "../../../react-auth0-wrapper";

it('renders without crashing', () => {
  const tree = renderer
    .create(<Auth0Context.Provider value={{isAuthenticated: true}}><Login/></Auth0Context.Provider>)
    .toJSON();
  expect(tree).toMatchSnapshot()
});
