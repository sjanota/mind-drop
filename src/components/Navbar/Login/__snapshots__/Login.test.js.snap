// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    Log out
  </button>
</div>
`;
